schemas/client.yml

```diff
@@ -59,6 +59,20 @@
       description: >
         Client domain.
 
+    - name: registered_domain
+      level: extended
+      type: keyword
+      short: The highest registered client domain, stripped of the subdomain.
+      description: >
+        The highest registered client domain, stripped of the subdomain.
+
+        For example, the registered domain for "foo.contoso.com" is "contoso.com".
```

Comment:
Please leave "google.com" as the example in each of the descriptions.

I want readers to be thinking about ECS and their context, not

> What's that contoso.com thing? And why does it look like a Microsoft website?